python/lsst/meas/astrom/astrometry.py

```diff
@@ -57,6 +59,18 @@ class AstrometryConfig(RefMatchConfig):
         default=0.001,
         min=0,
     )
+    doMagnitudeOutlierRejection = pexConfig.Field(
+        dtype=bool,
+        doc=("If True then a rough zeropoint will be computed from matched sources "
+             "and outliers will be rejected in the iterations."),
+        default=False,
+    )
+    magnitudeOutlierRejectionNSigma = pexConfig.Field(
+        dtype=float,
+        doc=("Number of sigma (measured from the distribution) in magnitude "
+             "for a potential match to be rejected during iteration."),
+        default=3.0,
```

Comment:
I think it would be good to be explicit in the docstring: "for a reference/source match to be rejected".